bika/lims/content/analysisrequest.py

```diff
@@ -1418,41 +1418,37 @@
             },
         )
     ),
-    # Another way to obtain a transition date is using getTransitionDate
-    # function. We are using a DateTimeField/Widget here because in some
-    # cases the user may want to change the Received Date.
-    # AnalysisRequest and Sample's DateReceived fields needn't to have
-    # the same value.
-    # This field is updated in workflow_script_receive method.
-    DateTimeField(
+    # This field is a mirror of a field in Sample with the same name
+    ProxyField(
         'DateReceived',
+        proxy="context.getSample()",
         mode="rw",
         read_permission=View,
         write_permission=ModifyPortalContent,
         widget=DateTimeWidget(
-            label=_("Date Received"),
+            label=_("Date Sample Received"),
+            show_time=True,
             description=_("The date when the sample was received"),
             visible={
                 'edit': 'visible',
                 'view': 'visible',
                 'header_table': 'visible',
-                'sample_registered': {
-                    'view': 'invisible', 'edit': 'invisible'},
-                'to_be_sampled': {'view': 'invisible', 'edit': 'invisible'},
-                'scheduled_sampling': {
-                    'view': 'invisible', 'edit': 'invisible'},
-                'sampled': {'view': 'invisible', 'edit': 'invisible'},
-                'to_be_preserved': {'view': 'invisible', 'edit': 'invisible'},
-                'sample_due': {'view': 'invisible', 'edit': 'invisible'},
+                'sample_registered': {'view': 'visible', 'edit': 'invisible'},
+                'to_be_sampled': {'view': 'visible', 'edit': 'invisible'},
+                'scheduled_sampling': {'view': 'visible', 'edit': 'invisible'},
+                'sampled': {'view': 'visible', 'edit': 'invisible'},
+                'to_be_preserved': {'view': 'visible', 'edit': 'invisible'},
+                'sample_due': {'view': 'visible', 'edit': 'invisible'},
                 'sample_prep': {'view': 'visible', 'edit': 'invisible'},
-                'sample_received': {'view': 'visible', 'edit': 'invisible'},
-                'attachment_due': {'view': 'visible', 'edit': 'invisible'},
+                'sample_received': {'view': 'visible', 'edit': 'visible'},
+                'attachment_due': {'view': 'visible', 'edit': 'visible'},
```

Comment:
Better to allow the edition at Sample View only. See my comment above.

Reply:
Done